.github/workflows/nix.yml

```diff
@@ -20,6 +21,15 @@ jobs:
       with:
         name: cofree-coffee
         authToken: '${{ secrets.CACHIX_AUTH_TOKEN }}'
-    
-    - name: Deploy 🚀
-      run: nix build
\ No newline at end of file
+
+    - name: Check 🔎
+      run: |
+        nix flake check
+
+    - name: Run Shell 🐚
+      run: |
+        nix develop
```

Comment:
Why run `nix develop`?

Reply:
a) to check that the dev environment isn't borken
b) to force CI to build the dev environment and then push it to cachix so that users can get cache hits

Reply:
oh nice. I hadn't thought of that.